client/me/purchases/list/index.jsx

```diff
@@ -13,8 +13,48 @@ import Main from 'components/main';
 import MeSidebarNavigation from 'me/sidebar-navigation';
 import PurchasesHeader from './header';
 import PurchasesSite from './site';
+import SimpleNotice from 'notices/simple-notice';
 
 const PurchasesList = React.createClass( {
+	renderNotice() {
+		const { noticeType } = this.props;
+
+		if ( ! noticeType ) {
+			return null;
+		}
+
+		let message, status;
+
+		if ( 'cancel-success' === noticeType ) {
+			message = this.translate(
+				'Your purchase was canceled and refunded. The refund may take up to ' +
+				'7 days to appear in your PayPal/bank/credit card account.'
+			);
+
+			status = 'is-success';
+		}
+
+		if ( 'cancel-problem' === noticeType ) {
+			message = this.translate(
+				'There was a problem canceling your purchase. ' +
```

Comment:
We already have also similar translation:
```js
'There was a problem canceling %(purchaseName)s. ' +
'Please try again later or contact support.',
```
Would it be possible to use only one of them?